packages/components/src/components/accordion/_accordion.scss

```diff
@@ -197,8 +196,7 @@
     }
 
     .#{$prefix}--accordion__arrow {
-      /* rtl:ignore */
-      transform: rotate(-90deg);
+      transform: rotate(-90deg) #{'/*rtl:rotate(-90deg)*/'};
```

Comment:
```suggestion
      transform: rotate(-90deg) #{'/*rtl:ignore*/'};
```

Reply:
Ah, right! I think I had tried that before realizing Sass was not reading the rtl comments unless interpolated. Thanks for catching that @asudoh 😄